sw/device/silicon_creator/lib/sigverify_rsa_key.h

```diff
@@ -49,7 +49,7 @@ typedef struct sigverify_rsa_key {
    */
   uint32_t n0_inv[8];
   /**
-   * Exponent.
+   * Exponent. For sigverify, the only exponent supported is 65537.
    */
   uint32_t exponent;
```

Comment:
Since we only support 65537 I think we can also remove this field. WDYT?

Reply:
Seems reasonable to me; will do.

Reply:
Looks like `exponent` is also a member of the manifest; should it be removed there too?

Reply:
Good point, but let's leave it for now since we would have to update a couple of other files, e.g. manifest.rs and probably the signer. we don't use it for signature verification in mask rom anyway. Would you mind creating an issue and assigning it to me? Thanks!